### ISSUE TITLE
[BUG] Token HUD not working after 1.3.0 update - t.getContent is not a function

### ISSUE THREAD

**Issue by jeffisowens** [Larkinabout/fvtt-token-action-hud-core#90]
**Describe the bug**
After updating, the HUD does not appear when a token is selected. In the console, I see the following error:

```
Uncaught (in promise) TypeError: undefined. t.getContent is not a function
[Detected 1 package: token-action-hud-core]
    at CompendiumActionHandler._getPlaylistEntries (token-action-hud-core.min.js:1:15240)
    at CompendiumActionHandler.getCompendiumEntries (token-action-hud-core.min.js:1:15164)
    at CompendiumActionHandler._getCompendiumActions (token-action-hud-core.min.js:1:14764)
    at CompendiumActionHandler.buildCompendiumActions (token-action-hud-core.min.js:1:14641)
    at async ActionHandler._buildCompendiumActions (token-action-hud-core.min.js:1:20564)
    at async Promise.all (:30000/index 2)
    at async ActionHandler.buildActionList (token-action-hud-core.min.js:1:17515)
    at async TokenActionHud._updateHud (token-action-hud-core.min.js:1:65254)
```

**Steps to reproduce**
1. Update to version 1.3.0
2. Load a world
3. Click on a token - HUD does not appear
4. Look in the console to see the error

**Versions:**
- Token Action HUD Core: 1.3.0
- Token Action HUD System: token-action-hud-pf2e v1.3.0
- System: Pathfinder Second Edition 4.10.4
- Foundry VTT: 10.291
- Browser: Google Chrome 111.0.5563.65

**Comment by cgplayer2000** [comment]
Just adding my addition to this bug -- I'm using Foundry V10.291 and DnD 5e 2.1.5
![image](https://user-images.githubusercontent.com/10798879/230449167-a81a13cc-1f33-46e5-b71d-e0c6d0f83847.png)

**Comment by Larkinabout** [comment]
Fixed in Token Action HUD Core 1.3.1.

**Comment by jeffisowens** [comment]
Thanks for the quick fix!
